gr-qtgui/lib/eye_sink_c_impl.h

```diff
@@ -70,6 +77,12 @@ class QTGUI_API eye_sink_c_impl : public eye_sink_c
                     QWidget* parent = NULL);
     ~eye_sink_c_impl() override;
 
+    // Disallow copy/move because of the raw pointers.
+    eye_sink_c_impl(const eye_sink_c_impl&) = delete;
+    eye_sink_c_impl& operator=(const eye_sink_c_impl&) = delete;
+    eye_sink_c_impl(eye_sink_c_impl&&) = delete;
+    eye_sink_c_impl& operator=(eye_sink_c_impl&&) = delete;
```

Comment:
The raw pointer you're referring to is the `QWidget* parent`, correct? Are there any others?

Reply:
Also `d_argv` and `d_main_gui`.

It could be that it's actually safe to copy `d_main_gui` and `d_argv`, as they are not destroyed locally (I assume QT owns destroying them?). `d_argv` may have to be solved some other way, though.